### ISSUE TITLE
FCGN or MGN training problem

### ISSUE THREAD

**Issue by xjtAlgo** [ZhangHanbo/Visual-Manipulation-Relationship-Network-Pytorch#8]
Hello,when I use the following commands:
    "python main.py --dataset vmrdcompv1 --frame fcgn --net res101 --cuda"
    "python main.py --dataset vmrdcompv1 --frame mgn --net res101 --cuda".
    I can see the decline in loss, and get a model.
    But when I apply this model to the test set, the accuracy rate is equal to zero.
    I print my output of the test set, and the predict of grasp box generated by each image is the same grasp box.
    The following are the prediction box parameters gained by fcgn test:
    # grasps -> array([[5.9643127e+02, 3.6898294e+02, 5.7293231e+02, 4.6866534e+02,
    #                  5.0924857e+02, 4.6022067e+02, 5.3274756e+02, 3.6053827e+02,
    #                  2.5008336e-01]], dtype=float32))
    It should be noted that I have no problem training and predicting on the VMRD dataset for the all_in_one model. 
    Why is model FCGN or model MGN training wrong?  I would be very grateful if you can reply me. Thanks!

**Comment by xjtAlgo** [comment]
You can set "self.use_objdet_branch= True" in MGN. It will run successfully.

**Comment by YiQi7** [comment]
I have also encountered the same problem, may I ask how fcgn should solve it
